jobs/JHAFS_WAV_PREP

```diff
@@ -33,6 +33,11 @@ if [ "${RUN_ENVIR^^}" != "NCO" ]; then
   source ${USHhafs}/hafs_pre_job.sh.inc
   export machine=${WHERE_AM_I:-wcoss2}
   module list
+  # A temporary workaround due to the hpc-stack version of wgrib2 does not work
+  # on Orion to convert grib2 files into netcdf format.
+  if [ ${machine} = orion ]; then
+    export WGRIB2=/apps/wgrib-2.0.8/wgrib2/wgrib2
```

Comment:
@mrinalbiswas and @BijuThomas-NOAA, will this /apps/wgrib-2.0.8/wgrib2/wgrib2 version works for all other HAFS jobs on Orion, if so, we may change the wgrib2 version in the modulefiles/hafs.orion.lua instead.

Reply:
@BinLiu-NOAA I can run with this version and let you know. The path on hafs.orion.lua directs wgrib2 to setenv("WGRIB2", "/apps/contrib/NCEP/libs/hpc-stack/intel-2018.4/impi-2018.4/wgrib2/2.0.8/bin/wgrib2"). I am not sure whether this is a working version of wgrib2 or not. The default on Orion points to: /apps/wgrib2-3.0.2/wgrib2/wgrib2.

Reply:
@mrinalbiswas, appreciate it if you could help to test if the /apps/wgrib2-3.0.2/wgrib2/wgrib2 version and its corresponding module version can work for the whole HAFS workflow on Orion. In such case, we can just implement this fix inside hafs.orion.lua. Thanks!

Reply:
@mrinalbiswas - I originally approved this PR before Bin's suggestion. I requested a re-review once this has been tested. Thanks!